### PR TITLE
Smoke test to unbind and bind service enabled.

### PR DESCRIPTION
**Pull request by RichardKnop**
# What

I have enabled the smoke test that tests unbinding
and binding of a service. It works now since the issue
in Postgres API has been fixed.

See https://github.com/tsuru/postgres-api/pull/14 for details.

I have also added rake to Gemfile as it was missing for some reason.
# How To Test

To test, please make sure you run Ansible to have your environment
up to date (newest version of Postgres API) and run smoke tests:

```
make test-gce DEPLOY_ENV=yourenvironment
```
# Who Should Test This

Anybody except @RichardKnop.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ gem 'rspec'
 gem 'serverspec', '~> 2.18.0'
 gem 'ansible_spec'
 gem 'highline'
-
+gem 'rake'
```
Comment (dcarley) [inline Gemfile:6]: Could you break this and the `Gemfile.lock` change into a separate commit which explains why we need to add this?

File: spec/endtoend/endtoend_spec.rb [hunk 2]
```diff
@@ -144,15 +144,7 @@
       expect(@workspace.tsuru_command.stdout).to include query
     end
 
-    it "should be able to connect to the applitation via HTTPS with a valid cert" do
```
Comment (dcarley) [inline spec/endtoend/endtoend_spec.rb:147]: Why did this move to the bottom? It should probably be next to the other HTTPS test, which it would replace, if anything.

Comment (RichardKnop) [inline spec/endtoend/endtoend_spec.rb:147]: I see. We thought since it's the only test which  is being skipped right now, it should be at the end of the test suite. We can move it back.

File: Gemfile (hunk 1 above)
Comment (RichardKnop) [inline Gemfile:6]: Done by Hector.

File: spec/endtoend/endtoend_spec.rb (hunk 2 above)
Comment (keymon) [inline spec/endtoend/endtoend_spec.rb:147]: Personally I like it like this, so the pending tests are at the end. Also, before it was not next to the HTTPS test, it had `should get log output in 3 seconds` before.

Comment (dcarley) [inline spec/endtoend/endtoend_spec.rb:147]: np. I think that it too would have benefited from being a separate commit saying that, then I wouldn't have needed to ask. But not a blocker for merging.

**Comment by dcarley** [comment]
Ansible pulled and deployed the new version of postgresapi automatically. Tests ran fine against my environment :+1:

File: spec/endtoend/endtoend_spec.rb (hunk 2 above)
Comment (keymon) [inline spec/endtoend/endtoend_spec.rb:147]: Thx for merging. 

True, should be in a different commit. Gone now :)
